### ISSUE TITLE
KubernetesInformerCreatorTest.testInformerInjection is flaky

### ISSUE THREAD

**Issue by brendandburns** [kubernetes-client/java#2013]
```
Error:  Failures: 
Error:    KubernetesInformerCreatorTest.testInformerInjection:139 No requests exactly matched. Most similar request was:  expected:<
GET
[path] /api/v1/pods

watch: false
> but was:<
GET
/api/v1/namespaces/default/configmaps?resourceVersion=0&watch=false

watch: false
>
```

**Comment by yue9944882** [comment]
/assign

@brendandburns interestingly, is the test getting flaky after a recent change?

**Comment by brendandburns** [comment]
@yue9944882 this is fixed (I hope) in #2012

I believe that adding Windows to the build matrix is lighting up some additional flakiness.

When I look at the tests, they are clearly written in a potentially flaky way. Please see #2012 for the fixes.

**Comment by yue9944882** [comment]
the flakiness involved by this test seems solved now

/close
